saturn-rapids/environment.yml

```diff
@@ -0,0 +1,31 @@
+name: saturn
+channels:
+- nvidia
+- rapidsai
+- numba
+- defaults
+- conda-forge
+dependencies:
+- blas=*=mkl
+- bokeh
+- cvxpy
+- dask-ml
+- dask=2.30.0
+- distributed=2.30.1
+- ipykernel
+- ipywidgets
+- matplotlib
+- numpy
+- pandas
+- pip
+- prefect
+- pyarrow
+- python=3.7
+- rapids=0.17.0=cuda10.1_py37_g180f433_165
+- s3fs
```

Comment:
`scikit-learn` should be in this image because rapids will utilize classes from there, like pipelines and others